README.md

```diff
@@ -105,6 +105,45 @@ e.g. `helm -n  laa-fala-staging delete el-123-fee-change`
+| Education                              | `edu`<br>/check?categories=edu     | [Find a legal aid adviser for education](https://staging.find-legal-advice.justice.gov.uk/check/education)                                                               |
+| Family                                 | `mat`<br>/check?categories=mat     | [Find a legal aid adviser for family](https://staging.find-legal-advice.justice.gov.uk/check/family)                                                                     |
+| Family Mediation                       | `fmed`<br>/check?categories=fmed   | [Find a legal aid adviser for family mediation](https://staging.find-legal-advice.justice.gov.uk/check/family-mediation)                                                 |
+| Housing                                | `hou`<br>/check?categories=hou     | [Find a legal aid adviser for housing](https://staging.find-legal-advice.justice.gov.uk/check/housing)                                                                   |
+| Housing Loss Prevention Advice Service | `hlpas`<br>/check?categories=hlpas | [Find a legal aid adviser for the Housing Loss Prevention Advice Service](https://staging.find-legal-advice.justice.gov.uk/check/housing-loss-prevention-advice-service) |
+| Immigration or Asylum                  | `immas`<br>/check?categories=immas | [Find a legal aid adviser for immigration or asylum](https://staging.find-legal-advice.justice.gov.uk/check/immigration-or-asylum)                                       |
+| Mental Health                          | `mhe`<br>/check?categories=mhe     | [Find a legal aid adviser for mental health](https://staging.find-legal-advice.justice.gov.uk/check/mental-health)                                                       |
+| Modern Slavery                         | `mosl`<br>/check?categories=mosl   | [Find a legal aid adviser for modern slavery](https://staging.find-legal-advice.justice.gov.uk/check/modern-slavery)                                                     |
+| Prison Law                             | `pl`<br>/check?categories=pl       | [Find a legal aid adviser for prison law](https://staging.find-legal-advice.justice.gov.uk/check/prison-law)                                                             |
+| Public Law                             | `pub`<br>/check?categories=pub     | [Find a legal aid adviser for public law](https://staging.find-legal-advice.justice.gov.uk/check/public-law)                                                             |
+| Welfare Benefits                       | `wb`<br>/check?categories=wb       | [Find a legal aid adviser for welfare benefits](https://staging.find-legal-advice.justice.gov.uk/check/welfare-benefits)                                                 |
```

Comment:
are we setting ourselves up for problems by exposing the staging url like this? I know that we do it elsewhere but this is a list of inviting URLs for a bot to scrape.

Could we show the URLs as just the slug e.g 
```https://staging.find-legal-advice.justice.gov.uk/check/welfare-benefits```
would be
```/check/welfare-benefits```
or 
```{HOST}/check/welfare-benefits```
I am thinking that in the main these are useful for devs and they will actually be looking for localhost:3000/check/welfare-benefits anyway.

Reply:
Good Point, have updated.